sunpy/visualization/animator/wcs.py

```diff
@@ -299,10 +303,30 @@ def plot_start_image_2d(self, ax):
 
         return im
 
+    def update_bounds(self):
+        """
+        Update vmin/vmax when set to 'auto'.
+        """
+        print(self.imshow_kwargs)
+        if 'vmax' in self.imshow_kwargs and self.imshow_kwargs['vmax'] == 'auto':
+            # set clip interval to 90% of maximum value
+            clip_interval = (1, 90)*u.percent
+            clip_percentages = clip_interval.to('%').value
+            vmin, vmax = AsymmetricPercentileInterval(*clip_percentages).get_limits(self.data_transposed)
+            return vmin, vmax
+        else:
+            return None
```

Comment:
```suggestion
```

Reply:
This is necessary because I'm calling this method without passing vmax='auto' would case an unpacking error

Reply:
I thought functions return none if you don't trigger a manual return?

Reply:
Yeah, sorry got a tad bit confused there. Making the changes now.